pkg/steps/artifacts.go

```diff
@@ -216,6 +216,27 @@ type PodClient interface {
 	podCmdExecutor
 }
 
+func waitForContainer(podClient PodClient, ns, name, containerName string) error {
+	logrus.Tracef("Waiting for container %s in pod %s in namespace %s", containerName, name, ns)
```

Comment:
nit:

```go
logrus.WithFields(logrus.Fields{
  "namespace": ns,
  "name": name,
  "container": containerName,
}).Trace("Waiting for container to be running.")
```

Reply:
My line is cribbed from [this master line][1], and `pkg/steps` has no `WithFields` consumers as of the current master (d9084dd58).  Can I punt on this and pick it up if master pivots `pkg/steps` to `WithFields` before this PR lands?

[1]: https://github.com/openshift/ci-tools/blob/d9084dd58eefba841b7e4e96bd843f6074e11f71/pkg/steps/artifacts.go#L220

Reply:
It would be faster to fix it, than to type all that out :)